docs/cli-options.rst

```diff
@@ -202,7 +202,7 @@ Instaloader to login.
    This feature requires the browser_cookie3 library.
    Compatible with :option:`--cookiefile` if you want to load cookies from browser profiles.
    Incompatible with :option:`--login` due to potential username mismatch between user input and browser login.
-   Supported browsers: Chrome, Firefox, Edge, Opera, Safari, Brave.
+   Supported browsers: Chrome, Chromium, Firefox, Edge, Brave, Opera, Opera GX, Safari, Vivaldi and Librewolf.
```

Comment:
I think we should sort the browsers somehow, e.g. alphabetically.

It is probably unclear that for Opera GX, 'opera_gx` must be specified as value.

Reply:
Makes sense. I've ordered the browsers in the `supported_browsers` variable as well for consistency.

I've also changed it to "Opera_GX".

Reply:
Nice, thanks!